pages/about.html

```diff
@@ -4,9 +4,32 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <title>Lilly</title>
+  <link href="styles/style.css" rel="stylesheet">
 </head>
-<body>
-  
-</body>
+  <body>
+    <header>
+      <h1 class="cover__name">Lilly Hebert</h1>
+    </header>
+    <nav>
+        <ul>
+            <li><a href="index.html">Home</a></li>
+            <li><a href="about.html">About</a></li>
+            <li><a href="portfolio.html">Portfolio</a></li>
+          </ul>
+    </nav>
+    <main>
+      <h2 class="about_headers">About Me</h2>
+      <p>Introduction paragraph</p>
+      <h2 class="about_headers">Personal Info</h2>
+      <div class="container">
+        <div>Hobbies</div>
+        <div>Work experience</div>
+        <div>Fun Facts</div>
+      </div>
```

Comment:
The `<div>` tag works here, but consider using something more meaningful, such as the `<section>` and `<article>` elements.